src/features/home/components/homeBlog.css

```
@@ -0,0 +1,73 @@
+.home-blog-container {
+  margin: 100px 100px;
```

이 경우에는 축약형을 쓸 수 있을 것 같습니다
```suggestion
  margin: 100px;
```